feat(rules): no-snapshots (#330)
jest-community/eslint-plugin-jest: docs/rules/no-snapshots.md

# Disallow snapshots (no-snapshots)

Jest’s snapshot feature allows you to assert that a value has not changed from a
stored value in a previous test. The matchers `toMatchSnapshot`,
`toMatchInlineSnapshot`, `toThrowErrorMatchingSnapshot` and
`toThrowErrorMatchingInlineSnapshot` will generate snapshots when used inside
test blocks.

## Rule Details

> Using snapshots will often result in poorly documented, difficult to debug tests

Reviewer: ```suggestion
Using snapshots may result in poorly documented, difficult to debug tests
```
